src/openssl/mod.rs

```diff
@@ -230,12 +230,11 @@ impl ECVRF {
     fn generate_nonce(&mut self, secret_key: &BigNum, data: &[u8]) -> Result<BigNum, Error> {
         // Bits to octets from data - bits2octets(h1)
         // Length of this value should be dependent on qlen (i.e. `SECP256k1` is 32)
```

Comment:
Maybe this line does not apply anymore as we do not need the "workaround" anymore. :)